### PR TITLE
Responsiveness count activity

### PR DESCRIPTION
**Pull request by MitchellQ**
This PR works towards #53

Made the count activity responsive on all devices and in portrait mode

other changes include
- created a helper method called isSmall in the screen class
- created a helper method called isLandscape in the screen class
these are mainly to help with readability
- renamed a file to be inline with dart best practices

### CODE REVIEW

File: lib/util/screen.dart [hunk 1]
```diff
@@ -22,6 +22,12 @@ class Screen {
   static bool isPortrait(BuildContext context) =>
       Screen.orientation(context) == Orientation.portrait;
 
+  static bool isLandscape(BuildContext context) =>
+      Screen.orientation(context) == Orientation.landscape;
```
Comment (jacksos101) [inline lib/util/screen.dart]: I feel like we can just use !isPortrait() but it's no big issue.

File: lib/views/activites/count_activity_view.dart [hunk 2]
```diff
@@ -23,153 +24,238 @@ class CountActivityView extends StatefulWidget {
...
-                                  count = count + 1;
-                                });
-                              }
-                            },
-                          ),
+                        child: Body(
+                          'Pass',
+                          size: Screen.width(context) >= 600
+                              ? 30
+                              : Screen.width(context) <= 350 ? 16 : 20,
```
Comment (jacksos101) [inline lib/views/activites/count_activity_view.dart]: Is this somewhere you'd want to use your Screen class methods?

File: lib/util/screen.dart (hunk 1 above)
Comment (jacksos101) [reply lib/util/screen.dart]: Or you could go `static bool isLandScape(BuildContext context) => !isPortrait(context);`

File: lib/widgets/activities/activity_app_bar.dart [hunk 3]
```diff
@@ -12,6 +13,7 @@ class ActivityAppBar extends StatelessWidget with PreferredSizeWidget{
       leading: Container(),
       title: Heading(
         text,
+        size: Screen.width(context) >= 600 ? 30 : 20,
```
Comment (jacksos101) [inline lib/widgets/activities/activity_app_bar.dart]: Should this be isTablet()?

**Comment by jacksos101** [comment]
Screenshot_20190823-165459

Actually I am getting some issues on my phone...

File: lib/views/activites/count_activity_view.dart (hunk 2 above)
Comment (MitchellQ) [reply lib/views/activites/count_activity_view.dart]: Yes it is

File: lib/widgets/activities/activity_app_bar.dart (hunk 3 above)
Comment (MitchellQ) [reply lib/widgets/activities/activity_app_bar.dart]: yes it should

File: lib/util/screen.dart (hunk 1 above)
Comment (MitchellQ) [reply lib/util/screen.dart]: I don't think your second comment makes any difference also I just found it easier to read that as I kept forgetting the ! or miss reading it

File: lib/views/activites/count_activity_view.dart (hunk 2 above)
Comment (MitchellQ) [reply lib/views/activites/count_activity_view.dart]: fixed

File: lib/widgets/fact_file/editAnswer.dart [hunk 4]
```diff
@@ -9,7 +9,7 @@ class EditAnswer extends StatelessWidget {
       "To edit your answer, re-scan the QR code.",
       style: TextStyle(
         fontSize:
-            Screen.isTablet(context) ? 25.0 : Screen.isSmall(context) ? 14 : 20,
+            Screen.isTablet(context) ? 25.0 : Screen.isSmall(context) ? 12 : 20,
```
Comment (jacksos101) [inline lib/widgets/fact_file/editAnswer.dart]: This wouldn't affect my phone as it's not less than 350 wide

Comment (MitchellQ) [reply lib/widgets/fact_file/editAnswer.dart]: surely not what is the logical pixels

**Comment by MitchellQ** [comment]
We changed the small device rule to include 360 which is sams device width which is called as a small phone

**Pull request merged**